### ISSUE TITLE
Load a data matching configuration not working in version 8

### ISSUE THREAD

**Issue by ecopets** [PrestaShop/PrestaShop#33705]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

When importing products in version 8 >  the section "Load a data matching configuration" does NOT load a saved configuration. 
 The SAVE button will save a configuration, but the LOAD button will not retrieve it.  
 Also, the DELETE button give a "Technical Error" but no cause.  
 These functions are extremely useful when importing large lists of products from different suppliers to avoid having to repeatedly map the same fields.
 
These functions work perfectly in version 1.7.8.10

### Expected behavior

The expectation is that the LOAD and DELETE buttons work properly.

### Steps to reproduce

in version 8 (particularly 8.1.1) got toe Product Import
select file and proceed thru the steps.
when  at the "Load a data matching configuration" 1- name a configuration and save it after mapping fields and BEFORE starting the import.  or 2- repeat the above and at the "Load a data matching configuration", try to LOAD a saved configuration.  the saved mapping SHOULD load, but in 8.1.1, nothing happens.

### PrestaShop version(s) where the bug happened

8.1.1

### PHP version(s) where the bug happened

8.0

### Your company or customer's name goes here (if applicable).

Eco Pet Supplies

**Comment by Hlavtox** [comment]
I confirm it doesnt work

**Comment by sallemiines** [comment]
Hello @ecopets 

Could you please provide us with a screen recorder for the above steps ! 
I didn't manage to reproduce your same issue ! 

Please check and feedback ! 

Thank you

**Comment by Hlavtox** [comment]
@sallemiines 
1. Go to BO > Import
2. Prepare csv with for example only ID, name and price
3. Start import
4. Assign the columns
5. Save this configuration
6. Go import this file again, try to load the configuration - doesnt do anything

**Comment by ecopets** [comment]
I concur. It is VERY simple!  No video/screen shot is needed. If you look at the database table, you can see the entry is saved and that the configuration would be what you set. But the load button DOES NOTHING.  It works in Prestashop 1.7.8.10 but not any release, that I am aware of, of 8.xx
If someone is unable to duplicate the issue and it works, then a) be sure you are some level of Prestashop version 8.  And verify what PHP version is in use.

**Comment by hibatallahAouadni** [comment]
Hello @ecopets 

Thanks @Hlavtox for your steps :pray: 
We are aware of this issue, it is already in our debug backlog. This issue is a duplicate of #32798. 
To be informed when it's fixed, please subscribe to the issue mentioned above.

Thanks!

**Comment by hibatallahAouadni** [comment]
Duplicate of #32798
